mappings/net/minecraft/entity/passive/PandaEntity.mapping

```diff
@@ -63,7 +63,7 @@ CLASS net/minecraft/class_1440 net/minecraft/entity/passive/PandaEntity
 		ARG 1 bitmask
 	METHOD method_6534 getScaredAnimationProgress (F)F
```

Comment:
I believe this method (plus a few that mention "scared", except `isScaredByThunderstorm`) also needs a change.